weapons.yaml

```diff
@@ -905,6 +907,20 @@ SonicZap:
 			Heavy: 80
 			Concrete: 60
 		DamageTypes: Prone50Percent, TriggerProne, ExplosionDeath
+		ValidTargets: Water, Underwater
+
+SquidPunch:
+	ReloadDelay: 32
+	Range: 1c856
+	Report: vsquat1a.wav
+	ValidTargets: Water, Underwater
+	Projectile: Bullet
+		Speed: 30
```

Comment:
This adds a weird delay in between the animation and the impact.
How about using a higher value (maybe `3000`, don't know if that's already overkill, though)?